pages/understanding-json-schema/_index.md

```diff
@@ -8,7 +8,7 @@ Understanding JSON Schema
 
 <img src="understanding/octopus.svg" className="w-1/2 float-right" />
 
-JSON Schema is a powerful tool for validating the structure of JSON
+[JSON](../learn/glossary#json) [Schema](../learn/glossary#schema) is a powerful tool for validating the structure of JSON
```

Comment:
JSON Schema is one single thing, please dont make this separation of terms.

Reply:
Okay I understand that they are one single thing but I did not find one single glossary term for JSON schema. JSON and schema are separate glossary term in glossary page. How do I point them to single glossary term ? Could you please give me idea about that?